app/Http/Controllers/Auth/AuthController.php

```diff
@@ -122,140 +125,236 @@ public function create(array $data)
 
     public function requestOtp(Request $request)
     {
-        $this->validate($request, [
-            'mobile' => 'required|numeric',
+        $request->validate([
+            'eid' => 'required|string',
         ]);
-        $newNumber = ltrim($request->newnumber, '0');
-        $newCode = $request->code;
-        $newCountry = $this->getNewCountry($newCode);
-        User::where('id', $request->id)->update(['mobile' => $newNumber, 'mobile_code' => $newCode, 'country' => $newCountry]);
+
         try {
-            $code = $request->input('code');
-            $mobile = ltrim($request->input('mobile'), '0');
-            $number = '(+'.$code.') '.$mobile;
-            $result = $this->sendOtp($mobile, $code);
-            $response = ['type' => 'success', 'message' => 'OTP has been sent to '.$number.'.Please Verify to Login'];
+            // Decrypt the email
+            $email = Crypt::decrypt($request->eid);
 
-            return response()->json($response);
-        } catch (\Exception $ex) {
-            $result = [$ex->getMessage()];
-            \Log::error('Error: '.$ex->getMessage());
+            // Find the user by email
+            $user = User::where('email', $email)->firstOrFail();
+
+            if (! $user) {
+                return errorResponse(__('message.user_does_not_exist'));
+            }
```

Comment:
firstOrfail() i think will throw exception if entry not found and hence will go in catch block, so you dont need to check user does not exist condition